includes/class-adst-plugins-stats-api.php

```diff
@@ -696,24 +806,24 @@ public function display_active_installs( $atts ) {
 		$plugins          = get_option( "_site_transient_bsf_tr_plugin_Active_Count_$wp_plugin_author" );
 
 		if ( '0' === $plugins ) {
-				return __( 'Please verify author slug.', 'wp-themes-plugins-stats' );
+			return __( 'Please verify author slug.', 'wp-themes-plugins-stats' );
 		} else {
 			if ( empty( $plugins ) || false === $plugins ) {
-					$plugins = $this->bsf_display_plugins_active_count( 'query_plugins', $api_params['plugin_author'] );
-				if ( 'Please verify author slug.' === $plugins || 'Please verify author slug.' === $plugins ) {
-					return __( 'Please verify author slug.', 'wp-themes-plugins-stats' );
+				$plugins = $this->bsf_display_plugins_active_count( 'query_plugins', $api_params['plugin_author'] );
+				if ( 'Please verify author slug.' === $plugins ) {
+					return __( 'Please verify author slug. ', 'wp-themes-plugins-stats' );
 				}
-					$num = $plugins;
-					$n   = $this->bsf_display_human_readable( $num );
-					return $n;
+				$num = $plugins;
+				$n   = $this->bsf_display_human_readable( $num );
+				return $n;
 			} else {
-					$plugins = $this->bsf_delete_active_count_transient( $wp_plugin_author );
+				$plugins = $this->bsf_delete_active_count_transient( $wp_plugin_author );
 				if ( null === $plugins ) {
 					return __( 'Please verify author slug.', 'wp-themes-plugins-stats' );
 				}
-					$num = $plugins;
-					$n   = $this->bsf_display_human_readable( $num );
-					return $n;
+				$num = $plugins;
+				$n   = $this->bsf_display_human_readable( $num );
```

Comment:
Rename $n variable name to something which will explain variable's purpose here.